tests/std/tests/P2442R1_views_chunk/test.cpp

```diff
@@ -197,12 +197,13 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
         }
     }
 
-    STATIC_ASSERT(CanMemberEnd<const R> == forward_range<const V>);
+    STATIC_ASSERT(CanMemberEnd<const R> == input_range<const V>);
     if constexpr (CanMemberEnd<const R>) {
         const same_as<sentinel_t<const R>> auto cs = as_const(r).end();
         assert((r.begin() == cs) == is_empty);
         STATIC_ASSERT(
-            common_range<R> == (common_range<const V> && (sized_range<const V> || !bidirectional_range<const V>) ));
+            common_range<
+                R> == (forward_range<const V> && common_range<const V> && (sized_range<const V> || !bidirectional_range<const V>) ));

```

Comment:
I ran into a problem that clang-format wants this line to look like this,, while Code Format Validation will complain the line more than 120 columns. Any ideas?

Reply:
The following are error messages from workflows:
[Expected formatting](https://dev.azure.com/vclibs/STL/_build/results?buildId=11717&view=logs&j=ce9c9df7-f6fa-59cf-0934-3339753bfc56&t=c6e68a6e-a0d3-5ff7-0941-3a725df1e075&l=20)
[Code Format Validation failed](https://dev.azure.com/vclibs/STL/_build/results?buildId=11719&view=logs&j=ce9c9df7-f6fa-59cf-0934-3339753bfc56&t=587b9e2b-1c42-5f9c-5a6a-63448075a96c&l=13)

Reply:
No problem, this is definitely an annoying issue. I second @miscco's recommendation https://github.com/microsoft/STL/pull/2878#discussion_r924459742 - adding an empty comment will force clang-format to wrap, and is typically the least intrusive way to fix this.

#2475 is our tracking issue for clang-format headaches like this, and they will reportedly get better in the next release of clang-format.

Reply:
I tried some other options to make this less ugly, but finally gave up and dropped the `// clang-format off` hammer.